aldryn_client/localdev/main.py

```diff
@@ -0,0 +1,85 @@
+import os
+import subprocess
+
+import click
+
+
+def create_workspace(client, website_slug, path=None):
+    website_git_path = 'git@github.com:aldryn/test-project.git'  # temporary
+
+    path = os.path.abspath(path) if path else None
+    website_id = client.get_website_id_for_slug(website_slug)
+
+    docker_compose_base = [
+        'docker-compose', '-f', os.path.join(path, 'docker-compose.yml')
```

Comment:
Did you think about requiring docker-compose in the requirements (and use it as a library)?